.github/workflows/test.yml

```diff
@@ -28,10 +28,24 @@ jobs:
           pip install pylint
           if [ -f setup.py ]; then pip install -e .; fi
           pip install transformers
+
+          #Install miniconda
+          wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+          bash Miniconda3-latest-Linux-x86_64.sh -b -u
```

Comment:
fyi I'm not sure how to make this work with localhost benchmarking. I tried and got weird errors. Can you wait for #104 to merge and then see if your stuff still works?

Reply:
This works well locally as long as `source ~/miniconda3/bin/activate` is used

Reply:
I tried that but got hit with
```
| ERROR: Package 'mlagility' requires a different Python: 3.10.9 not in '==3.8.*'
```